rabbitmq_amqp_python_client/entities.py

```diff
@@ -149,7 +149,26 @@ class ExchangeToExchangeBindingSpecification:
     binding_key: Optional[str] = None
 
 
-class StreamOptions:
+class StreamFilterOptions:
+    values: Optional[list[str]] = (None,)
+    match_unfiltered: bool = (False,)
```

Comment:
```suggestion
    match_unfiltered: bool = False
```

I don't think you need parenthesis there. The type hint is bool, so the default should be `False` instead of `(False,)`, which is a tuple (not a bool).